WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -41,18 +42,50 @@ final class DashboardViewController: UIViewController {
         return label
     }()
 
+    /// A stack view to display `storeNameLabel` with additional margins
+    ///
+    private lazy var innerStackView: UIStackView = {
+        let view = UIStackView()
+        view.layoutMargins = UIEdgeInsets(top: 0, left: Constants.leadingMargin, bottom: 0, right: 0)
+        view.isLayoutMarginsRelativeArrangement = true
+        return view
+    }()
+
+    /// A stack view for views displayed between the navigation bar and content (e.g. store name subtitle, top banner)
+    ///
     private lazy var stackView: UIStackView = {
```

Comment:
Do you think it might be necessary to update the name of this property to `mainStackView` or `headerStackView` to make it clearer?